Dockerfile

```
@@ -14,7 +14,7 @@ ARG GO_BUILD_TAGS
 
 COPY pkg ./pkg
 COPY cmd ./cmd
-RUN --mount=type=cache,target="/root/.cache/go-build" CGO_ENABLED=1 CGO_CFLAGS="-O2 -Wno-return-local-addr" \
+RUN --mount=type=cache,target="/root/.cache/go-build" CGO_ENABLED=0 \
```

```suggestion
RUN --mount=type=cache,target="/root/.cache/go-build" \
```
CGO_ENABLED=0 is default, you need to set it only when you want to enable it.